src/locales/hi.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Sorry, but these are not the instructions i was looking for. i should have referred to InstructionPage.vue specifically in the Issue.

We do not use FormInput or FormResults in Quiz Engine anymore.

Reply:
THANK YOU for the review. I am working towards the required changes. 
Sorry for the confusion. 
Just to clarify that only "General Instructions" part need the required changes or 
" Answering a Question: "  also?
Thank You
![image](https://github.com/user-attachments/assets/bdcf8248-5363-4f3a-8ce1-290121bf1e6c)